### ISSUE TITLE
Is there any form of alert testing?

### ISSUE THREAD

**Issue by MostHated** [TwiN/gatus#693]
### Describe the feature request

I was simply wondering if there was any sort alert testing for once you have configured an alert provider? 

I setup Telegram and Gotify, but don't exactly have a great way to test them aside from taking an existing monitor and forcing it to fail. 
It would be nice if there was perhaps a UI button to sent a test alert, or an API endpoint to hit specifically for sending a test message.
 If this already exists, I didn't see mention of it in the readme or on the site's alerting section.

The example below simply sends a generic message to your configured alert provider so you can be sure it is configured properly.

<details>
<summary>Telegram Example from Uptime Kuma</summary>

![](https://img.instance.id/OxySx1U3oaxh/direct)

</details>

Thanks,
-MH

### How long have you been using this project?

3 days

**Comment by tarocjsu** [comment]
Just build a docker/container, and deply a nginx web browser, than monitor it, shutdown it....the alert notify should be work....

**Comment by MostHated** [comment]
Yeah, I am aware that there are 'round-about' ways to do that, but I would like to request a simple 'Send Test' mechanism so that those extra steps are not needed.

**Comment by Shinmera** [comment]
Even without modifying the UI or API endpoints, there could simply be a command line command to send a test alert without having to muck with the rest of the config.

**Comment by TwiN** [comment]
The easiest way is to just add a condition that will fail, such as `[RESPONSE_TIME] < 1`.
